app/src/main/res/layout/fragment_home.xml

```diff
@@ -4,7 +4,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingHorizontal="10dp">
+    android:paddingHorizontal="10dp"
+    android:scrollbarSize="0dp">
```

Comment:
It can be better if the scrollbar of `ScrollView` is hidden which make no difference in functionality and looks good when hidden (scrollbar of scrollview)

Reply:
Okay. helps to know the reason of this change.